django/views/debug.py

```diff
@@ -26,7 +26,9 @@
     libraries={'i18n': 'django.templatetags.i18n'},
 )
 
-CURRENT_DIR = Path(__file__).parent
+
+def _builtin_template_path(name):
```

Comment:
Not sure this needs the `_` 🤷

Reply:
I didn't want to introduce any public API in a "cleanup/optimization" PR.

Reply:
This hook is used only in `ExceptionReporter`, so I would change this to a `ExceptionReporter` method.

> I didn't want to introduce any public API in a "cleanup/optimization" PR.

API is not considered public if it's not documented.

Reply:
> This hook is used only in `ExceptionReporter`

It's also used in `technical_404_response` and `default_urlconf`. I'll hold off making it an `ExceptionReporter` method unless you confirm you still want me to do that.

> API is not considered public if it's not documented.

I'll get rid of the underscore.

Reply:
> I'll get rid of the underscore.

Done: 9adc5e75db5bfa2b91956424b0b349f7f178787e